magma/backend/coreir/coreir_transformer.py

```diff
@@ -361,9 +370,21 @@ def __init__(self, backend, opts, decl):
         self.coreir_module = None
 
     def run_self(self):
-        self.coreir_module = self.run_self_impl()
-
-    def run_self_impl(self):
+        self.coreir_module = self._run_self_impl()
+        # Set symbol table data.
+        # NOTE(rsetaluri): For some reason, fetching the CoreIR names, re-orders
```

Comment:
Note: read note here. something is broken downstream making this not right for me.

Reply:
Can you try this change: https://github.com/rdaly525/coreir/pull/985